### ISSUE TITLE
Typescript - plugins do not have declaration files

### ISSUE THREAD

**Issue by DBaker85** [greensock/GSAP#323]
Hello, I was using the previous version of GSAP with a React / Typescript project.

I tried switching to GSAP3 and was trying to use the Motion path plugin, but none of the plugin files have a declaration.

`import * as MotionPathPlugin from "gsap/MotionPathPlugin"` gives an error.

I have to manually add a declaration to my global types (which is fine for now), but I think this is going to be a problem for future users who decided to switch to GSAP 3 with Typescript.

I think `import {MotionPathPlugin} from "gsap"` would be an even nicer implementation

**Comment by OSUblake** [comment]
We are still working on the definitions, and should have revised version in the next couple of weeks.

For now, just declare any missing types yourself.

```ts
declare var MotionPathPlugin: any;
```

Or ignore the errors for now.
```ts
// @ts-ignore
MotionPathPlugin.getRawPath();
```


> import * as MotionPathPlugin from "gsap/MotionPathPlugin" gives an error.

You should import like this:
```ts
import MotionPathPlugin from "gsap/MotionPathPlugin";
gsap.registerPlugin(MotionPathPlugin);
```
Or:
```ts
import { MotionPathPlugin } from "gsap/MotionPathPlugin";
gsap.registerPlugin(MotionPathPlugin);
```


> I think import {MotionPathPlugin} from "gsap" would be an even nicer implementation

If you use webpack or rollup, do this.
```ts
import { gsap, MotionPathPlugin } from "gsap/all";
gsap.registerPlugin(MotionPathPlugin);
```

**Comment by DBaker85** [comment]
Thanks for the answer. 
I did get it working with local declarations. I Just wanted to point out a few type errors that I came across. 
Thanks for the work

**Comment by jackdoyle** [comment]
Should be resolved in the latest release (3.0.2)
